problemsets-mat223/modules/module1-exercises.tex

```diff
@@ -126,6 +126,52 @@
     		    \item Yes.
 		    \end{enumerate}
 		\end{solution}
+		
+		\prob
+		    Use set notations, vectors, or linear combinations of vectors to describe the following sets.
+		    \begin{enumerate}
+		        \item The set of vectors with rational components in $\R^2$.
+		        \item The set of vectors with irrational components in $\R^2$.
+		        \item The set that contains the empty set.
+		        \item $\R^3$.
```

Comment:
What are you hoping for an answer for this one?

Reply:
I was hoping that students will consider linearly independent/basis vectors and use linear combinations to express this set. I agree this is too general and there are many acceptable answers...

Reply:
Yeah. Let's take it away.